### ISSUE TITLE
Failed to download glog-0.3.5.tar.gz

### ISSUE THREAD

**Issue by ansarikhurshid786** [facebook/react-native#21190]
I have created project in mac in android running fine but i am getting error to run in ios. 

Error console log as below: 
Failed to successfully download 'glog-0.3.5.tar.gz'.  Debug info:
ls: /Users/mac1/.rncache/glog-0.3.5.tar.gz: No such file or directory
shasum: /Users/mac1/.rncache/glog-0.3.5.tar.gz:
Command PhaseScriptExecution failed with a nonzero exit code

I am behind proxy so i think not allowing to download from github. 

is there any solution or work around ?

**Comment by phoa** [comment]
I just encountered this problem. I'm behind proxy too. Looks like proxy setup is required to whatever command line tool is used by react-native to download the file.

However, I can manually download the required .tar.gz files from web browser.
Here's what I did.

- Download the .tar.gz files. The url to download the files can be found at the bottom of file `node_modules/react-native/scripts/ios-install-third-party.sh`

- Manually copy and paste all the .tar.gz files to `~/.rncache`

- run `react-native run-ios`. This time it should be able to pick up the file from `~/.rncache` folder, and unpacked them.
